package.json

```diff
@@ -565,6 +595,12 @@
         "command": "vscode-docker.api.configure",
         "title": "Add Docker files to Workspace (API)"
       },
+      {
+        "command": "vscode-docker.ACR-queueBuild",
+        "title": "Queue Build",
```

Comment:
Shouldn't this be Azure-specific?  E.g. "Queue ACR Build" and "Queue an Azure build from a Dockerfile"?

Reply:
Actually, I'm not sure "queue" makes sense at all.  This uses the Quick Build feature, right?  Shouldn't it be something like "Azure Quick Build"?

Reply:
Well, in the previous API we used a function called queue build, so it was to be consistent. Right now we have something call scheduled run, but that is a function that can do queue build and more.

Reply:
I just talk with my team, and they agree on Quick Build because it is only doing that for this specific scenario. So I'll be changing the name